components/lightsnapcast/player.c

```diff
@@ -1307,21 +1296,14 @@ static void player_task(void *pvParameters) {
   audio_set_mute(scSet.muted);
 
   // wait for early time syncs to be ready
+  xSemaphoreTake(latencyBufFullSemaphoreHandle, portMAX_DELAY);
```

Comment:
aren't too late pcm chunks accumulating if you block here? Or did I miss something?

Reply:
pcm chunks are discarded in `insert_pcm_chunk` until the buffer is full.